apps/frontend/src/app/Components/AmpReactionModeText.tsx

```diff
@@ -17,16 +17,16 @@ export const ampReactionMap = {
   }
 } as const
 const sqBadgeStyle = { mx: 0.25, px: 0.25 }
-export default function AmpReactionModeText({ reaction, trigger }: { reaction: AmpReactionKey, trigger?: Variant }) {
+export default function AmpReactionModeText({ reaction, trigger }: { reaction: AmpReactionKey, trigger?: "cryo" | "pyro" | "hydro" }) {

```

Comment:
can you make a type, instead of repeating it multiple times? Looks like that type would be used in other places too

`type AmpReactionTrigger = "cryo" | "pyro" | "hydro"

Reply:
Probably doesnt occur often enough to matter. we have too many weird types around.

Reply:
if we have to change this later, i blame you :)